src/models/oracle-api-key.model.ts

```diff
@@ -56,7 +56,7 @@ export class OracleApiKey extends Entity {
       default: false,
     },
   })
-  expired = false;
+  expired: boolean = false;
```

Comment:
when you have litteral types (`boolean`, `number`, `string`) that are initialized, there's no need to explicitely set the type.
this could stay `expired = false;` 
TS will correctly infer that it's a `boolean`.